diff --git a/include/keplerian_toolbox/lambert_problem.hpp b/include/keplerian_toolbox/lambert_problem.hpp
index 0efe867..0b41889 100644
--- a/include/keplerian_toolbox/lambert_problem.hpp
+++ b/include/keplerian_toolbox/lambert_problem.hpp
@@ -36,6 +36,11 @@
 namespace kep_toolbox
 {
 
+class KEP_TOOLBOX_DLL_PUBLIC lambert_problem;
+
+// Streaming operator for the class kep_toolbox::lambert_problem.
+KEP_TOOLBOX_DLL_PUBLIC std::ostream &operator<<(std::ostream &, const lambert_problem &);
+
 /// Lambert Problem
 /**
  * This class represent a Lambert's problem. When instantiated it assumes a prograde orbit (unless otherwise stated)
@@ -57,7 +62,7 @@ class KEP_TOOLBOX_DLL_PUBLIC lambert_problem
     static const array3D default_r2;
 
 public:
-    friend std::ostream &operator<<(std::ostream &, const lambert_problem &);
+    friend KEP_TOOLBOX_DLL_PUBLIC std::ostream &operator<<(std::ostream &, const lambert_problem &);
     lambert_problem(const array3D &r1 = default_r1, const array3D &r2 = default_r2,
                     const double &tof = boost::math::constants::pi<double>() / 2, const double &mu = 1.,
                     const int &cw = 0, const int &multi_revs = 5);
@@ -111,9 +116,6 @@ private:
     int m_multi_revs;
 };
 
-// Streaming operator for the class kep_toolbox::lambert_problem.
-KEP_TOOLBOX_DLL_PUBLIC std::ostream &operator<<(std::ostream &, const lambert_problem &);
-
 } // namespace kep_toolbox
 
 #endif // KEP_TOOLBOX_LAMBERT_PROBLEM_H
