diff --git a/include/keplerian_toolbox/lambert_problem.hpp b/include/keplerian_toolbox/lambert_problem.hpp
index 0efe867..0b41889 100644
--- a/include/keplerian_toolbox/lambert_problem.hpp
+++ b/include/keplerian_toolbox/lambert_problem.hpp
@@ -36,6 +36,11 @@
 namespace kep_toolbox
 {
 
+class KEP_TOOLBOX_DLL_PUBLIC lambert_problem;
+
+// Streaming operator for the class kep_toolbox::lambert_problem.
+KEP_TOOLBOX_DLL_PUBLIC std::ostream &operator<<(std::ostream &, const lambert_problem &);
+
 /// Lambert Problem
 /**
  * This class represent a Lambert's problem. When instantiated it assumes a prograde orbit (unless otherwise stated)
@@ -57,7 +62,7 @@ class KEP_TOOLBOX_DLL_PUBLIC lambert_problem
     static const array3D default_r2;
 
 public:
-    friend std::ostream &operator<<(std::ostream &, const lambert_problem &);
+    friend KEP_TOOLBOX_DLL_PUBLIC std::ostream &operator<<(std::ostream &, const lambert_problem &);
     lambert_problem(const array3D &r1 = default_r1, const array3D &r2 = default_r2,
                     const double &tof = boost::math::constants::pi<double>() / 2, const double &mu = 1.,
                     const int &cw = 0, const int &multi_revs = 5);
@@ -111,9 +116,6 @@ private:
     int m_multi_revs;
 };
 
-// Streaming operator for the class kep_toolbox::lambert_problem.
-KEP_TOOLBOX_DLL_PUBLIC std::ostream &operator<<(std::ostream &, const lambert_problem &);
-
 } // namespace kep_toolbox
 
 #endif // KEP_TOOLBOX_LAMBERT_PROBLEM_H
diff --git a/include/keplerian_toolbox/sims_flanagan/spacecraft.hpp b/include/keplerian_toolbox/sims_flanagan/spacecraft.hpp
index cdfa3fc..c5841ba 100644
--- a/include/keplerian_toolbox/sims_flanagan/spacecraft.hpp
+++ b/include/keplerian_toolbox/sims_flanagan/spacecraft.hpp
@@ -36,6 +36,10 @@ namespace kep_toolbox
 namespace sims_flanagan
 {
 
+class KEP_TOOLBOX_DLL_PUBLIC spacecraft;
+
+KEP_TOOLBOX_DLL_PUBLIC std::ostream &operator<<(std::ostream &s, const spacecraft &in);
+
 /// Spacecraft
 /**
  * A container for system design parameters of a spacecraft.
@@ -45,7 +49,7 @@ namespace sims_flanagan
 
 class KEP_TOOLBOX_DLL_PUBLIC spacecraft
 {
-    friend std::ostream &operator<<(std::ostream &s, const spacecraft &in);
+    friend KEP_TOOLBOX_DLL_PUBLIC std::ostream &operator<<(std::ostream &s, const spacecraft &in);
 
 public:
     spacecraft() : m_mass(0), m_thrust(0), m_isp(0) {}
@@ -95,7 +99,6 @@ private:
     double m_isp;
 };
 
-KEP_TOOLBOX_DLL_PUBLIC std::ostream &operator<<(std::ostream &s, const spacecraft &in);
 } // namespace sims_flanagan
 } // namespace kep_toolbox
 
